Edits/New Content - General, Java, Git, R (#826)
Codecademy/docs: content/java/concepts/strings/strings.md

---
Title: 'Strings'

> Description: 'A String in Java is an object that holds a sequence of characters contained within a pair of double quotes (").'

Reviewer: I don't believe string should be capitalized. I understand that in Java code the type is declared as 'String' but we generally discuss items in docs generically: referring to the class of object in lieu of the code. I reviewed the Python and Javascript pages and they both refer to strings generically

  - 'paths/computer-science'
---

> A **`String`** in Java is an object that holds a sequence of characters contained within a pair of double quotes (`"`). It is not a primitive datatype.

Reviewer: ```suggestion
Strings in Java are objects that can hold a sequence of characters contained within a pair of double quotes (`"`). It is not a primitive datatype.
```

> `String`s can either be compared by value via method (e.g., [`.equals()`](https://www.codecademy.com/resources/docs/java/strings/equals)) or by reference, or location in memory, (e.g., `==`) via operator.

Reviewer: ```suggestion
Strings can either be compared by value via method (e.g., [`.equals()`](https://www.codecademy.com/resources/docs/java/strings/equals)) or by reference, or location in memory, (e.g., `==`) via operator.
```

## Example

Java strings provide a way to store text such as words, sentences, or whole paragraphs. They can be any length and may contain letters, numbers, symbols, and spaces:

```java
import java.util.*;

class StringExample {
  public static void main(String[] args) {

>     // Using a String literal

Reviewer: ```suggestion
    // Using a string literal
```
